OpenEdXMobile/res/layout/course_detail_field.xml

```diff
@@ -14,13 +13,12 @@
         android:layout_marginBottom="@dimen/edx_margin"
         android:orientation="horizontal">
 
-        <com.joanzapata.iconify.widget.IconImageView
+        <androidx.appcompat.widget.AppCompatImageView
```

Comment:
We could provide it a static start drawable: `ic_dashboard`. Because according to the code (`CourseDetailFragment Line# 166`) we are always providing the same icon.

Reply:
We could provide it a static start drawable: `ic_dashboard`. Because according to the code (`CourseDetailFragment Line# 166`) we are always providing the same icon.

Reply:
As per discussion with khalid, we can ignore this file changes